src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,45 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String color;
+    private int horsepower;
+    private String mark;
+
+    public Bulldozer(String color, int powerEngene, String mark) {
+        this.color = color;
+        this.horsepower = powerEngene;
+        this.mark = mark;
+    }
+
     public Bulldozer() {
```

Comment:
remove default constructors, you don't need them in your classes

Reply:
in tests default designers are caused

Reply:
sorry, my bad, forgot about it)